opendebates/templates/opendebates/vote.html

```diff
@@ -7,131 +7,128 @@
-                </g>
-                <g>
-                  <path fill="#010101"
-                        d="M0,0v41.9h25l9-9V0H0z M31,29.9h-9v9H3V3h28V29.9z"/>
-                </g>
+                <rect x="7" y="6.1" fill="#010101" width="20.1" height="3.3"/>
               </g>
-            </svg>
-          </a>
-        </div>
-        <div>
-          <a class="citation" target="_blank" href="{{ idea.citation }}">
-            <strong>{% blocktrans %}Citation:{% endblocktrans %}</strong>
-          </a>
-          <a target="_blank" href="{{ idea.citation }}">{{ idea.citation }}</a>
-        </div>
+              <g>
+                <rect x="7" y="12.1" fill="#010101" width="20.1" height="3.3"/>
+              </g>
+              <g>
+                <rect x="7" y="18.1" fill="#010101" width="20.1" height="3.3"/>
+              </g>
+              <g>
+                <rect x="7" y="24.1" fill="#010101" width="20.1" height="3.3"/>
+              </g>
+              <g>
+                <rect x="7" y="30.1" fill="#010101" width="11.1" height="3.3"/>
+              </g>
+              <g>
+                <path fill="#010101"
+                      d="M0,0v41.9h25l9-9V0H0z M31,29.9h-9v9H3V3h28V29.9z"/>
+              </g>
+            </g>
+          </svg>
+        </a>
       </div>
-    {% endif %}
-
-    {% if related1 %}
-    <div class="idea-related">
-      <h4>{% blocktrans %}You may also be interested in...{% endblocktrans %}</h4>
-      <div class="row">
-        <div class="col-md-6">
-          {% include "opendebates/snippets/idea.html" with idea=related1 is_related=1 %}
-        </div>
-        <div class="col-md-6">
-          {% if related2 %}
-          {% include "opendebates/snippets/idea.html" with idea=related2 is_related=1 %}
-          {% endif %}
-        </div>
+      <div>
+        <a class="citation" target="_blank" href="{{ idea.citation }}">
+          <strong>{% blocktrans %}Citation:{% endblocktrans %}</strong>
+        </a>
+        <a target="_blank" href="{{ idea.citation }}">{{ idea.citation }}</a>
       </div>
     </div>
-    {% endif %}
+  {% endif %}
 
-    {% comment %}
-      {% if comment_list|length %}
-      <div class="comment-count">
-        <h4>
-          {% if comment_list|length == 1 %}
-          {{ comment_list|length }} Comment
-          {% else %}
-          {{ comment_list|length }} Comments
-          {% endif %}
-        </h4>
+  {% if related1 %}
+  <div class="idea-related">
+    <h4>{% blocktrans %}You may also be interested in...{% endblocktrans %}</h4>
+    <div class="row">
+      <div class="col-md-6">
+        {% include "opendebates/snippets/idea.html" with idea=related1 is_related=1 %}
       </div>
+      <div class="col-md-6">
+        {% if related2 %}
+        {% include "opendebates/snippets/idea.html" with idea=related2 is_related=1 %}
+        {% endif %}
+      </div>
+    </div>
+  </div>
+  {% endif %}
 
-      <div class="comment-list">
+  {% comment %}
```

Comment:
this is all commented-out because we're not doing comments.  (confusing, I know!)  You can just rip it all out if you want; it'll still be in git history.